packages/terra-site/src/examples/date-time-picker/Index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-date-time-picker/docs/README.md';
+import { version } from 'terra-date-time-picker/package.json';
+
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import DateTimePickerSrc from '!raw-loader!terra-date-time-picker/src/DateTimePicker';
+// Example Files
+import DateTimePickerDefault from './DateTimePickerDefault';
+import DateTimePickerDefaultDateTime from './DateTimePickerDefaultDateTime';
+import DateTimePickerDefaultDateOnly from './DateTimePickerDefaultDateOnly';
+import DateTimePickerExcludeDates from './DateTimePickerExcludeDates';
+import DateTimePickerIncludeDates from './DateTimePickerIncludeDates';
+import DateTimePickerFilterDates from './DateTimePickerFilterDates';
+import DateTimePickerMinMax from './DateTimePickerMinMax';
+import DateTimePickerDST from './DateTimePickerDST';
+import DateTimePickerInsideModal from './DateTimePickerInsideModal';
+
+const DateTimePickerExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
+    <Markdown id="readme" src={ReadMe} />
+    <PropsTable id="props" src={DateTimePickerSrc} />
+    <h2 id="default">Default</h2>
+    <DateTimePickerDefault />
+    <h2 id="default-date-time">Default Date Time</h2>
+    <DateTimePickerDefaultDateTime />
+    <h2 id="default-date-only">Default Date Only</h2>
+    <DateTimePickerDefaultDateOnly />
+    <h2 id="exclude-dates">Exclude Dates</h2>
+    <DateTimePickerExcludeDates />
+    <h2 id="filter-dates">Filter Dates</h2>
+    <DateTimePickerFilterDates />
+    <h2 id="include-dates">Include Dates</h2>
+    <DateTimePickerIncludeDates />
+    <h2 id="min-max">Min Max Selection Options</h2>
+    <DateTimePickerMinMax />
+    <h2 id="dst">Daylight Saving</h2>
+    <p>
+      The missing hour occurs when DST begins at these recent past and near future dates between 2:00 and 3:00.
```

Comment:
Are these specific to the US or world wide?

Reply:
These daylight saving dates are specific to the US. I updated the message to call out that these are specific to the US and added an additional link for daylight saving dates for other time zones. https://github.com/cerner/terra-core/pull/768/commits/3763caa7628bc29b961a755d2d760b627a28bbfa